mora/service/orgunit.py

```diff
@@ -205,6 +205,16 @@ def get_one_orgunit(c, unitid, unit=None,
     elif details is UnitDetails.FULL:
         unittype = common.get_uuid(rels['enhedstype'][0], required=False)
 
+        if rels['overordnet'][0]['uuid']is not None:
+            parent = get_one_orgunit(c, rels['overordnet'][0]['uuid'],
```

Comment:
Since we're obtaining the parent, we might as well include it in the output, since it might be useful for other stuff as well. So just assign this to `r['parent']` instead.